forge-gui/src/main/java/forge/gamemodes/limited/LimitedPlayer.java

```diff
@@ -33,6 +34,9 @@ public class LimitedPlayer {
     private static final int NobleBanneretActive = 1 << 6;
     private static final int PalianoVanguardActive = 1 << 7;
     private static final int GrinderRemoveFromPool = 1 << 8;
+    private static final int SearcherNoteNext = 1 << 9;
+    private static final int WhispergearBoosterPeek = 1 << 10;
+    private static final int IllusionaryInformantPeek = 1 << 11;
 
     private static final int MAXFLAGS = CantDraftThisRound | ReceiveLastCard | CanRemoveAfterDraft | SpyNextCardDrafted
```

Comment:
MAXFLAGS purpose?

Reply:
Yea i had it in originally when i worked on this a few years ago, but i'm starting to think its not actually helpful. Might need to turn these into a long soon though.